### PR TITLE
 Add me-central-1, ELB Logging S3 Bucket Policy, Local Zones, Display Names 

### PR DESCRIPTION
**Pull request by Nuru**
## what
- Add new `me-central-1` zone
- Add missing Local Zones from `us-east-1` and `us-west-2`
- Add ELB Logging S3 Bucket Policy output
- Add region display name map
- Update minimum Terraform version to 0.14.0

## why
- Maintain complete coverage of regions
- Improve coverage of local regions
- To enable ELB access logging to an S3 bucket, you have always needed a region-specific bucket policy. This module provides the account number to use in a policy template based on region, but that is no longer sufficient. There are now 3 different templates for S3 bucket policies, and again which one to use depends on the region. Therefore, to further simplify the task for others, this module can output the full policy JSON, not just the account number. 
- There is no official API to convert from the official region name, like `eu-north-1`, to a human-friendly name, like "Europe (Stockholm)". This module now outputs a map of such names, derived from the AWS Java SDK. 
- Previous minimum Terraform version 0.13.0 has different formatting, causing formatting checks to fail when compared to code formatted by later versions

## references
- https://aws.amazon.com/blogs/aws/now-open-aws-region-in-the-united-arab-emirates-uae/
- https://github.com/aws/aws-sdk-java/issues/1467
- Supersedes and closes #18 
- Supersedes and closes #19

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -72,29 +72,55 @@ introduction: |-
...
   and strictly algorithmically derived so that people can more easily interpret them. The `short` region
   code abbreviations typically match the prefix of the Availability Zone IDs in that region, but this is
-  not guaranteed.
+  not guaranteed. The `short` abbreviations for local regions are generally of the form AWS uses, with
+  the region prefix and dashes removed.
   - The `fixed` abbreviations are always exactly 3 characters for regions and 4 characters
   for availability zones and local zones, but have some exceptional cases (China, Africa, Asia-Pacific South, US GovCloud)
-  that have non-obvious abbreviations.
+  that have non-obvious abbreviations. If a future new region causes a conflict with an established local zone
+  abbreviation, we may change the local zone abbreviation to to keep the region mappings consistent. For example,
```
Comment (aknysh) [inline README.yaml]:
```suggestion
  abbreviation, we may change the local zone abbreviation to keep the region mappings consistent. For example,
```

**Review by Benbentwo — DISMISSED** [review]
Nice Readme 🙏

File: examples/complete/outputs.tf [hunk 2]
```diff
@@ -13,6 +13,28 @@ output "short" {
   value       = local.code_maps.to_short[var.region]
 }
 
+output "identity_size" {
+  description = "Size of identity map"
+  value       = local.identity_size
+}
```
Comment (aknysh) [inline examples/complete/outputs.tf:19]: add empty lines b/w the outputs

**Review by aknysh — COMMENTED** [review]
a few nitpicks

**Review by Benbentwo — DISMISSED** [review]
LGTM

**Pull request merged**